engine/class_modules/sc_death_knight.cpp

```diff
@@ -2655,6 +2657,7 @@ struct death_knight_action_t : public Base
       gain = this -> player -> get_gain( util::inverse_tokenize( this -> name_str ) );
     }
 
+    // These both need review to see if the spec aura is applying or not, if it is, this will double dip
```

Comment:
Only a boolean is set here, it shouldn't double dip and the worse case scenario is one superfluous set of test and operation is being run once on spell construction. But i can check later